mix.exs

```diff
@@ -27,7 +27,8 @@ defmodule SweetXml.Mixfile do
 
   defp deps do
     [
-      {:ex_doc, ">= 0.0.0", only: :dev, runtime: false}
+      {:ex_doc, ">= 0.0.0", only: :dev, runtime: false},
+      {:dialyxir, "~> 1.4", only: [:dev, :test], runtime: false}
```

Comment:
We're not keen on having `dialyxir` as a dependency just yet.

We might in the future, it is just not something that we want to do in isolation, in this repo only.
We currently are in the process of cleaning up our repos, and decided that this kind of change would be discussed down the line.

We'll gladly add those later on if it is something we decide to do as an organization.